File: node.js/백준/Bronze/10809. 알파벳 찾기/알파벳 찾기.js
const fs = require('fs');
const input = fs.readFileSync('/dev/stdin').toString().trim();

const alphabetList = "abcdefghijklmnopqrstuvwxyz".split("");

const inputArr = input.split("");

for (let i = 0; i < alphabetList.length; i++) {
  if (inputArr.includes(alphabetList[i])) {
    alphabetList[i] = inputArr.indexOf(alphabetList[i]);
  } else {
    alphabetList[i] = -1;
  }
}

console.log(alphabetList.join(" "));

File: Python/백준/Bronze/10809. 알파벳 찾기/알파벳 찾기.py
alphabet_list = list("abcdefghijklmnopqrstuvwxyz")
input_arr = list(input())

for i in range(len(alphabet_list)):
    if alphabet_list[i] in input_arr:
        alphabet_list[i] = input_arr.index(alphabet_list[i])
    else:
        alphabet_list[i] = -1

print(*alphabet_list, sep=" ")